openadapt/signals.py

```diff
@@ -238,7 +222,8 @@ def __access_url_signal(self, http_url):
         Read signal data from an HTTP URL.
         """
         # Get the signal from the URL.
-        response = requests.get(http_url, allow_redirects=True)
+        HEADERS = {'User-Agent': 'Mozilla/5.0 (iPad; CPU OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148'}
```

Comment:
Split across multiple lines (use `black`) 🙏

Reply:
Resolved in [0adf43d](https://github.com/MLDSAI/OpenAdapt/pull/160/commits/0adf43d4a585ba2cf5d3850c9856a0af5646ab58)